libs/common/domain/src/lib/model/record/metadata.model.ts

```diff
@@ -174,7 +174,6 @@ export interface DatasetSpatialExtent {
 export interface DatasetTemporalExtent {
   start?: Date
```

Comment:
it looks like this is actually a required prop, since a time instant will be written using that field